appserver/admingui/devtests/run_test.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -ex
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+#
+# The contents of this file are subject to the terms of either the GNU
+# General Public License Version 2 only ("GPL") or the Common Development
+# and Distribution License("CDDL") (collectively, the "License").  You
+# may not use this file except in compliance with the License.  You can
+# obtain a copy of the License at
+# https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
+# or packager/legal/LICENSE.txt.  See the License for the specific
+# language governing permissions and limitations under the License.
+#
+# When distributing the software, include this License Header Notice in each
+# file and include the License file at packager/legal/LICENSE.txt.
+#
+# GPL Classpath Exception:
+# Oracle designates this particular file as subject to the "Classpath"
+# exception as provided by Oracle in the GPL Version 2 section of the License
+# file that accompanied this code.
+#
+# Modifications:
+# If applicable, add the following below the License Header, with the fields
+# enclosed by brackets [] replaced by your own identifying information:
+# "Portions Copyright [year] [name of copyright owner]"
+#
+# Contributor(s):
+# If you wish your version of this file to be governed by only the CDDL or
+# only the GPL Version 2, indicate your decision by adding "[Contributor]
+# elects to include this software in this distribution under the [CDDL or GPL
+# Version 2] license."  If you don't indicate a single choice of license, a
+# recipient has the option to distribute your version of this file under
+# either the CDDL, the GPL Version 2 or to extend the choice of license to
+# its licensees as provided above.  However, if you add GPL Version 2 code
+# and therefore, elected the GPL Version 2 license, then the option applies
+# only if the new code is made subject to such option by the copyright
+# holder.
+
+list_test_ids(){
+  echo admingui_all
+}
+
+merge_junit_xmls(){
+  JUD_DIR=$1
+  JUD=$WORKSPACE/results/junitreports/test_results_junit.xml
+  rm -f ${JUD} || true
+  echo "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" >> ${JUD}
+  echo "<testsuites>" >> ${JUD}
+  find ${JUD_DIR} -name "*.xml" -type f -exec cat '{}' \; | ${SED} 's/<?xml version=\"1.0\" encoding=\"UTF-8\" ?>//g' >> ${JUD}
+  echo -e "\n</testsuites>" >> ${JUD}
+}
+
+test_run(){
+  touch /tmp/password.txt
+  chmod 600 /tmp/password.txt
+  echo "AS_ADMIN_PASSWORD=" > /tmp/password.txt
+  echo "AS_ADMIN_NEWPASSWORD=admin" >> /tmp/password.txt
```

Comment:
In GUI dev tests if secure admin is on, user name and passwords are hardcoded. We need these values if secure admin is on. That is the reason Kokil added these lines.